osu.Game.Rulesets.Taiko/Difficulty/TaikoPerformanceAttributes.cs

```diff
@@ -20,6 +20,9 @@ public class TaikoPerformanceAttributes : PerformanceAttributes
         [JsonProperty("effective_miss_count")]
         public double EffectiveMissCount { get; set; }
 
+        [JsonProperty("estimated_ur")]
+        public double? EstimatedUR { get; set; }
```

Comment:
```suggestion
        public double? EstimatedUr { get; set; }
```
Needed for code quality to pass